torch/onnx/symbolic.py

```diff
@@ -599,17 +602,46 @@ def softplus(g, self, beta, threshold):
     return g.op('Softplus', self)
 
 
+def get_pool_ceil_padding(input, kernel_size, stride, padding):
+    dim = input.type().sizes()[-len(padding):]
+    ceiled_output_dim = [math.ceil((dim[i] + 2 * padding[i] - kernel_size[i]) / stride[i]) + 1
```

Comment:
math.ceil will turn the type to float. Here, and actually, the division difference between python 2 and 3: https://stackoverflow.com/questions/21316968/division-in-python-2-7-and-3-3